entity.go

```diff
@@ -81,6 +82,10 @@ type serializationDecoder struct {
 	dec *gob.Decoder
 }
 
+type nilType struct {
+	GoonNilValue string
+}
+
```

Comment:
Let's cut down on allocations and create a reusable value of this: `var nilValue = nilType{}`.

Reply:
Fixed 0ffa7caaada1ace69f367ee36d201bcebed76bbd